modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/batch/engine/resource/VulcanBatchEngineExportTaskResourceImpl.java

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.vulcan.internal.batch.engine.resource;
+
+import com.liferay.headless.batch.engine.resource.v1_0.ExportTaskResource;
+import com.liferay.portal.kernel.model.Company;
+import com.liferay.portal.kernel.model.User;
+import com.liferay.portal.vulcan.accept.language.AcceptLanguage;
+import com.liferay.portal.vulcan.batch.engine.resource.VulcanBatchEngineExportTaskResource;
+
+import javax.servlet.http.HttpServletRequest;
+
+import javax.ws.rs.core.MultivaluedMap;
+import javax.ws.rs.core.UriInfo;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Carlos Correa
+ */
+@Component(service = VulcanBatchEngineExportTaskResource.class)
+public class VulcanBatchEngineExportTaskResourceImpl
+	implements VulcanBatchEngineExportTaskResource {
+
+	@Override
+	public Object postExportTask(String name, String callbackURL, String fields)
+		throws Exception {
+
+		_exportTaskResource.setContextAcceptLanguage(_contextAcceptLanguage);
+		_exportTaskResource.setContextCompany(_contextCompany);
+		_exportTaskResource.setContextHttpServletRequest(
+			_contextHttpServletRequest);
+		_exportTaskResource.setContextUriInfo(_contextUriInfo);
+		_exportTaskResource.setContextUser(_contextUser);
+
+		return _exportTaskResource.postExportTask(
+			name, "JSON", callbackURL,
```

Comment:
@4lejandrito native batch engine points can export/import to format set by `ContentType` header parameter. I checked Import code and it looks that these virtual batch endpoints operates only with JSON.
If so, we can document it in documentation.